src/probnum/problems/zoo/filtsmooth/_filtsmooth_problems.py

```diff
@@ -0,0 +1,470 @@
+    :math:`r_n \sim \mathcal{N}(0, R)` for process noise covariance matrix :math:`Q`
+    and measurement noise covariance matrix :math:`R`.
+
+    Parameters
+    ----------
+    measurement_variance
+        Marginal measurement variance.
+    process_diffusion
+        Diffusion constant for the dynamics
+    times
+        Time grid for the filtering/smoothing problem.
+    initrv
+        Initial random variable.
+
+    Returns
+    -------
+    statespace_components
+        Dictionary containing
+        - dynamics model
+        - measurement model
+        - initial random variable
+
+    Notes
+    -----
+    No ``RegressionProblem`` object is returned. The dynamics model has to be linearized
+    in order to generate data.
+
+    """
+
+    def f(t, x):
+        return np.tanh(x)
+
+    def df(t, x):
+        return 1.0 - np.tanh(x) ** 2
+
+    def l(t):
+        return process_diffusion * np.ones(1)
+
+    initmean = np.zeros(1)
+    initcov = 3.0 * np.eye(1)
+    initrv = randvars.Normal(initmean, initcov)
+    dynamics_model = statespace.SDE(dimension=1, driftfun=f, dispmatfun=l, jacobfun=df)
+    measurement_model = statespace.DiscreteLTIGaussian(
+        state_trans_mat=np.eye(1),
+        shift_vec=np.zeros(1),
+        proc_noise_cov_mat=measurement_variance * np.eye(1),
+    )
+
+    statespace_components = dict(
+        dynamics_model=dynamics_model,
+        measurement_model=measurement_model,
+        initrv=initrv,
+    )
+    return statespace_components
+
+
+def logistic_ode(
+    y0: Optional[Union[np.ndarray, FloatArgType]] = None,
+    solver_initrv: Optional[randvars.RandomVariable] = None,
+    timespan: Optional[Tuple[float, float]] = None,
+    params: Optional[Tuple[float, float]] = None,
+    evlvar: Optional[Union[np.ndarray, FloatArgType]] = None,
+    ek0_or_ek1: Optional[int] = None,
+):
+    r"""Filtering/smoothing setup for a probabilistic ODE solver based on the logistic ODE.
+
+    This state space model puts an integrated Brownian motion prior on the dynamics
+    and constructs the ODE likelihood based on the vector field defining the
+    logistic ODE.
+
+    Parameters
+    ----------
+    y0
+        Initial conditions of the Initial Value Problem
+    solver_initrv
+        Initial random variable of the probabilistic ODE solver
+    timespan
+        Time span of the problem
+    params
+        Parameters for the logistic ODE
+    evlvar
+        See :py:class:`probnum.diffeq.GaussianIVPFilter`
+    ek0_or_ek1
+        See :py:class:`probnum.diffeq.GaussianIVPFilter`
+
+    Returns
+    -------
+    logistic_ivp
+        The initial value problem based on the logistic ODE.
+    statespace_components
+        Dictionary containing
+        - dynamics model
+        - measurement model
+        - initial random variable
+
+    See Also
+    --------
+    :py:class:`probnum.diffeq.GaussianIVPFilter`
+
+    """
+
+    if timespan is None:
+        timespan = (0.0, 2.0)
```

Comment:
do you prefer this style? because `timespan=(0., 2.)` would've worked just as well (tuples are immutable, if I remember correctly)